README.md

```diff
@@ -35,105 +33,29 @@ jobs:
         with:
           python-version: "3.8"
       - name: flake8 Lint
-        uses: reviewdog/action-flake8@v2
+        uses: rickstaa/action-flake8-composite@v1
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-### With non-default settings
-
-```yaml
-name: flake8 Lint
-
-on: [push, pull_request]
-
-jobs:
-  flake8-lint:
-    runs-on: ubuntu-latest
-    name: Lint
-    steps:
-      - name: Check out source repository
-        uses: actions/checkout@v2
-      - name: Set up Python environment
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.8"
-      - name: flake8 Lint
-        uses: reviewdog/action-flake8@v2
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          reporter: github-pr-check
-          level: warning
-          ignore: "F401"
-          exclude: "src/ignoreme.py"
-          max_line_length: "100"
-          path: "src"
-```
-
 See the Inputs section below for details on the defaults and optional configuration settings.
 
 ## Inputs
 
-Configure the Action with the following _optional_ settings:
-
-### Pip options
-
-#### `update_pip`
-
-**Optional**. Update `pip` before the flake8 install. Options: `[true, false]`. Defaults to `false`.
-
-#### `install_deps`
-
-**Optional**. Install the python project and dependencies (uses `setup.py`, `pyproject.toml` or `requirements.txt`) before the flake8 install. Options: `[true, false]`. Defaults to `false`.
-
-#### `dev_install`
-
-**Optional**. Install the project in editable `-e` mode (i.e.  [setuptools](https://pypi.org/project/setuptools/) 'develop mode'). Options: `[true, false]`. Defaults to `false`.
-
-#### `req_file_path`
-
-**Optional**. Path pointing to the requirements file. Defaults to `./requirements.txt`.
-
-### Flake8 options
-
-#### `flake8_version`
-
-**Optional**. flake8 version for testing. Options: `[latest, master, <VERSION NUMBER>]`. Defaults to `latest`.
+### `github_token`
 
-- `latest` = current PyPI release version
-- `master` = current [GitLab source repository master branch version](https://gitlab.com/pycqa/flake8)
-- `<VERSION NUMBER>` = the version number of the [flake8 PyPI package](https://pypi.org/project/flake8/) (e.g., `3.7.9`)
+**Required**. The [GITHUB_TOKEN](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow). Must be in form of `github_token: ${{ secrets.github_token }}`. Defaults to `${{ github.token }}`.
 
-#### `path`
+### `workdir`
 
-**Optional**. The path to the Python source file(s) or directory. Defaults to `"."`.
+**Optional**. The directory to run remark-lint in. Defaults to `.`.
 
-#### `args`
+#### `flake_args`
```

Comment:
Keep it consistent with action.yml and entrypoint.sh

Reply:
Very good catch thanks a lot.